index.bs

```diff
@@ -1824,18 +1824,24 @@ Charter Refinement</h3>
 		and their resolutions tracked in a disposition of comments
 		highlighting any issues not resolved by consensus.
 
-	The [=charter refinement=] phase concludes
-	when there is a [=Team Decision=]
+	Before the end of the announced duration
+	for the [=charter refinement=] phase,
+	the [=Team=]
 	(informed by the work of the [=Chartering Facilitator=])
-	to either complete charter refinement by initiating [=AC Review=] of the [=charter draft=],
-	or abandon the proposal.
-
-	The [=Team=] <em class=rfc2119>may</em> [=Team Decision|decide=]
-	to extend the charter refinement period
-	by sending an announcement with the same visibility as the initial [=charter review notice=].
+	<em class=rfc2119>must</em> [=Team Decision|decide=] which of the following to do:
+	* Complete charter refinement by initiating [=AC Review=] of the [=charter draft=].
+	* Abandon the proposal.
+	* Extend the charter refinement period.
+
+	The [=Team=] <em class=rfc2119>must</em> announce its [=Team Decision|decision=]
+	with the same visibility as the initial [=charter review notice=],
+	and <em class=rfc2119>must</em> include a rationale
+	if the decision is not to start the [=AC Review=].
```

Comment:
This reads a bit awkwardly / programmer-like to me. Here is an attempted simplification while maintaining precision, with re-use of the precise phrase "initiating [=AC Review=]" from the list for more explicit clarity. Editor's discretion to reject, incorporate, or iterate:
```suggestion
	if they are not initiating [=AC Review=].
```